Add Images for Watch & Follow (fixes #764)

JLKwong opened the pull request:
<!-- This is a new pull request template for treehouses.github.io.

Please make sure to:
- add (fixes #issue_number) to the end of pull request title when applicable,
- drop a link to your new pull request in our gitter chat.

Thank you for contributing! -->

<!-- issue number this pull request resolves -->
Fixes #764 

### Description
Added example images for watching and following in Step 0 (edited images with emphasis).

### Raw.Githack preview link
https://raw.githack.com/JLKwong/JLKwong.github.io/764-Images-Watch-Follow/#!pages/vi/firststeps.md

Commit:
Add images for watch and follow (fixes #764)

Commit:
Add images for watch and follow (fixes #764)

rjpadilla commented:
You may want to crop those images. The images take too much space and makes the page feel bloated. Cropping the image to only the area of interest will make it easier on the eyes.

Commit:
Add images for watching and following (fixes #764)

JLKwong commented:
> You may want to crop those images. The images take too much space and makes the page feel bloated. Cropping the image to only the area of interest will make it easier on the eyes.

I cropped the images and pushed a commit with the revised images. 😌👍

Commit:
Add images for watch and follow (fixes #764)

JLKwong commented:
I moved the cropped images to the `images` folder, changed the reference Markdown to point to the new location, and committed. Everything should be good for review.
